opendbc/car/toyota/carcontroller.py

```diff
@@ -233,6 +241,12 @@ def update(self, CC, CS, now_nanos):
         # Along with rate limiting positive jerk above, this greatly improves gas response time
         # Consider the net acceleration request that the PCM should be applying (pitch included)
         net_acceleration_request_min = min(actuators.accel + accel_due_to_pitch, net_acceleration_request)
+
+        # Prevents releasing brakes while stopping by considering engine creep force
+        # TODO: should this be if CS.neutral_accel > 0 instead?
+        if CS.acc_braking:
+          net_acceleration_request_min -= CS.neutral_accel
```

Comment:
can use actuatorsOutput, but we want to eventually not include the rate limit for better start from stop perf so that's a non starter

Reply:
this also regresses start from stop performance, fix that